src/Shared.Core/Chara/CharacterApi.cs

```diff
@@ -91,6 +91,20 @@ public static ControllerRegistration GetRegisteredBehaviour(Type controllerType,
         /// <typeparam name="T">Type with your custom logic to add to a character</typeparam>
         /// <param name="extendedDataId">Extended data ID used by this behaviour. Set to null if not used. Needed to copy the data in some situations.</param>
         public static void RegisterExtraBehaviour<T>(string extendedDataId) where T : CharaCustomFunctionController, new()
+        {
+            RegisterExtraBehaviour<T>(extendedDataId, 1000);
+        }
+
+        /// <summary>
+        /// Register new functionality that will be automatically added to all characters (where applicable).
+        /// Offers easy API for saving and loading extended data, and for running logic to apply it to the characters.
+        /// All necessary hooking and event subscribing is done for you. All you have to do is create a type
+        /// that inherits from <code>CharaExtraBehaviour</code> (don't make instances, the API will make them for you).
+        /// </summary>
+        /// <typeparam name="T">Type with your custom logic to add to a character</typeparam>
+        /// <param name="extendedDataId">Extended data ID used by this behaviour. Set to null if not used. Needed to copy the data in some situations.</param>
+        /// <param name="Priority">Defaults to 1000, Decrease to increase priority or vice versa such on <see cref="CharaCustomFunctionController.OnReload(GameMode)"/> or related functions.</param>
+        public static void RegisterExtraBehaviour<T>(string extendedDataId, int Priority = 1000) where T : CharaCustomFunctionController, new()
```

Comment:
Priority shouldn't have a default value when there's already an overload without it because it's unclear which overload will be used.